crt_portal/cts_forms/templates/vot_resources.html

```diff
@@ -46,55 +47,67 @@ <h1 id="get-help">
+            <a class="external-link--blue" href="tel:911">
+            {% trans "911" %}
+            </a>,
+            {% trans "the public should contact the Department of Justice" %} </li>
+
+            <li>{% trans "Election-crime complaints should be directed to your local" %}
+            <a class="external-link--blue" href="https://www.justice.gov/usao/find-your-united-states-attorney">U.S. Attorney's Offices </a>. {% trans "or the local" %}
+            <a class="external-link--blue" href="https://www.fbi.gov/contact-us">FBI office</a>
+            </li>
+          </ul>
           </p>
+        </section>
+        <section id="polling-place">
+          <h2>{% trans "Find the election laws, voting hours, or polling place locations in your state" %}</h2>
 
           <p>
-            {% trans "EXAMPLES" %}
+            {% trans "Go to" %} <a class="external-link--blue" href="https://usa.gov/election-office">usa.gov/election-office</a>  {% trans " to find your state election website." %}
           </p>
-          <ul>
-            <li>{% trans "Discrimination on the basis of race, color, or membership in a language minority gorup while voting or attempting to vote;" %}</li>
-            <li>{% trans "Voter registration issues;" %}</li>
-            <li>{% trans "Denial or lack of accessibility while voting;" %}</li>
-            <li>{% trans "Military or overseas absentee voting issues." %}</li>
-          </ul>
         </section>
+
         <section id="voting-rights">
-          <h2>{% trans "Learn about my voting rights" %}</h2>
+          <h2>{% trans "Learn about your voting rights" %}</h2>
 
           <p>
             <ul>
-            <li>{% trans "Want to know your federal rights? Download the Justice Department's new " %}<a class="link--blue" href="https://www.justice.gov/file/1517936/download">Know Your Voting Rights</a> guide.
-            </li>
-            <li>{% trans "Do you have a criminal conviction? Wondering if you can vote in your state?<br>Download the Justice Department's new " %}<a class="link--blue" href="https://www.justice.gov/voting/file/1507306/download">Guide to State Voting Rights that Apply After Criminal Conviction</a>.</li>
+            <li>{% trans "Find additional information about voting rights laws at " %}<a class="link--blue" href="https://justice.gov/crt/voting-section">www.justice.gov/crt/voting-section</a>.</li>
+
             <li>{% trans "Learn about federal laws that protect voters with disabilities at " %}<a class="link--blue" href="https://ada.gov">ada.gov</a>.</li>
-            <li>{% trans "Find additional information about voting rights at " %}<a class="link--blue" href="https://justice.gov">justice.gov</a>.</li>
+
+
+            <li>{% trans "Download the Justice Department's new " %}<a class="link--blue" href="https://www.justice.gov/file/1517936/download">Know Your Voting Rights</a> guide.
```

Comment:
should "Know your voting rights" be translated?

Reply:
i think so.